frontend/src/views/Game.tsx

```diff
@@ -52,12 +60,25 @@ function GamePage() {
    */
   useBeforeunload(() => 'Leaving this page may cause you to lose your current code and data.');
 
+  /**
+   * Display the notification as a callback from the notification
+   * subscription. Do not display anything if notification is already
+   * present or initiator is the current user.
+   */
+  const displayNotification = useCallback((result: Message) => {
+    const notificationResult: GameNotification = JSON.parse(result.body);
+    if (currentUser && notificationResult
+      && notificationResult.initiator
+      && currentUser.userId !== notificationResult.initiator.userId) {
```

Comment:
Possibly simplified logic:

```suggestion
    if (currentUser?.userId !== notificationResult?.initiator?.userId) {
```

Reply:
Applied -- thanks!